### ISSUE TITLE
Extensions we use and maintain

### ISSUE THREAD

**Issue by eadesignro** [DavidLambauer/awesome-magento2#4]
Hi,
I finally had a little time and i get back to you with some free extension we use and maintain:

https://github.com/EaDesgin/magento2-pdf-generator2 

https://community.magento.com/t5/Magento-2-Feature-Requests-and/Invoice-shipment-credit-memo-PDFs-Remove-Zend-Pdf-replace-with/idi-p/45879

Free extension using mPDF library to generate pdf from html. 

https://github.com/EaDesgin/magento2-warehouses

Free extension to solve the issue (at least partially ) https://community.magento.com/t5/Magento-2-Feature-Requests-and/Multinode-Inventory/idi-p/46810

https://github.com/EaDesgin/magento2-mobiledetect

The extension we use to separate mobile and desktop themes.

The last 2 ware created after creating some custom development.
We wold rely use some community feedback. Let me know what you think.

**Comment by DavidLambauer** [comment]
Hi @eadesignro, 

thanks for your recommendation, I will have a look on these (hopefully tonight) and come back with feedback.

**Comment by DavidLambauer** [comment]
Hi @eadesignro, I finally came to a small review. As you can see in the contribution Guide, I would like to see some minor changes. Most things are common and not a big deal to "fix" or "improve". It just makes sure, that everybody knows the purpose of the extension and knows how to install it. 

Due to that, I created a small PR (https://github.com/EaDesgin/magento2-pdf-generator2/pull/3) where I added three new sections to the Readme. 

A battle proofed and standard way to make things more clear is to use badges. You might want to checkout http://shields.io/ to generate some for your Repositories. 

The contribution guide also points to a point, where code should be tested. I do not force contributors to add tests, but I would love to see the major parts of the extension tested. 

Please check out the [Contribution Guide](https://github.com/DavidLambauer/awesome-magento2/blob/master/contributing.md). 

This is, what I would say, a best practice extension for the M2 Awesome List. You might want to check it out:

- https://github.com/yireo/Yireo_Whoops
- https://github.com/ebizmarts/magento2-magemonkey

In Summary: 

- Readme Improvements (Badges can help)
- Add some Unit or Integration Tests to make sure, the core of the Functionality works well. If this would be done later, it's ok. I will do my best to also add a few PR's to your Extensions!

Thanks for your recommendations. 


PS: I love your Videos! This is what makes Magento 2 Awesome!

**Comment by eadesignro** [comment]
Hi,

Thanks for the effort and time spent. We can close the issue now. 

P.S. During the spring we are swamped with work but unit testing is must!
